shared/cmd/flags.go

```diff
@@ -27,135 +41,49 @@ var (
 	// VerbosityFlag defines the logrus configuration.
 	VerbosityFlag = &cli.StringFlag{
 		Name:  "verbosity",
-		Usage: "Logging verbosity (trace, debug, info=default, warn, error, fatal, panic)",
+		Usage: "Verbosity of logs to display (trace, debug, info=default, warn, error, fatal, panic)",
 		Value: "info",
 	}
 	// DataDirFlag defines a path on disk.
 	DataDirFlag = &cli.StringFlag{
 		Name:  "datadir",
-		Usage: "Data directory for the databases and keystore",
+		Usage: "Specify a directory for the client database",
 		Value: DefaultDataDir(),
 	}
 	// EnableTracingFlag defines a flag to enable p2p message tracing.
 	EnableTracingFlag = &cli.BoolFlag{
 		Name:  "enable-tracing",
-		Usage: "Enable request tracing.",
-	}
-	// TracingProcessNameFlag defines a flag to specify a process name.
-	TracingProcessNameFlag = &cli.StringFlag{
-		Name:  "tracing-process-name",
-		Usage: "The name to apply to tracing tag \"process_name\"",
+		Usage: "Enable p2p message tracing.",
```

Comment:
This isn't limited to p2p. 
```suggestion
		Usage: "Enable process tracing via OpenTracing / Jaeger exporter.",
```